deps/llvm.mk

```diff
@@ -175,6 +175,19 @@ LLVM_CPPFLAGS += -flto
 LLVM_LDFLAGS += -flto
 endif # LLVM_LTO
 
+ifeq ($(BUILD_CUSTOM_LIBCXX),1)
+LLVM_LDFLAGS += -Wl,-R$(build_libdir)
+LLVM_CXXFLAGS += -I$(build_includedir)
+LLVM_CPPFLAGS += -I$(build_includedir)
+LLVM_CFLAGS += -I$(build_includedir)
+LLVM_LIBCXX_FLAGS := -lc++ -lc++abi
```

Comment:
Ah, reading further on ahead, we keep them separate so we can bootstrap `libcxx` without having it link to itself.  Nevermind this then.  I would just add the `LD` into the name to make it easy to understand what it is right off the bat.